pages/admin/index.tsx

```diff
@@ -207,7 +207,7 @@ export default function Requests() {
             </MenuList>
           </Menu> */}
         </Flex>
-        <Box border="1px solid" borderColor="border.secondary" borderRadius="12px">
+        <Box border="1px solid" borderColor="border.secondary" borderRadius="12px" bgColor="white">
```

Comment:
We need to add `bgColor="white"` to the Permit Holders and Admin Management tables as well